modules/setup-dev-environment/pages/migrate-a-svn-repository-to-github.adoc

```diff
@@ -9,13 +9,13 @@ For Subscription editions only.
 
 This page explains how to migrate a *SVN* repository and its history to a *Git* repository on https://github.com/[GitHub].
 
-== Prerequistes
+== Pre-requisites
```

Comment:
https://english.stackexchange.com/questions/500696/pre-requisite-vs-prerequisite

Reply:
It's Prerequisites :p